code-challenges.test.js

```diff
@@ -21,9 +21,35 @@ const people = [
 ]
 // Expected output: ["Ford Prefect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]
 
+describe("sentence", () => {
+  it("returns array sentence with name capitalized", () => {
+    expect(sentence(people)).toEqual(["Ford Perfect is a hitchhiker.", "Zaphod Beeblebrox is president of the galaxy.", "Arthur Dent is a radio employee."]);
+  })
+})
+
 
 // b) Create the function that makes the test pass.
 
+//declare function
+//takes in an array as parameter
+//iterate over the array using .map
+//concat
+//string interpolation
+
+const sentence = (personArray) => {
+  return personArray.map(value => {
+    let name = person.name.split(" ")
+    let firstLetter = name[0].slice(0, 1).toUpperCase()
+    let firstName = name[0].slice(1)
+    let lastLetter = name[0].slice(0, 1).toUpperCase()
+    let lastName = name[0].slice(1)
+  })
+}
+
+
+
+
+//what exaclty is .map mapping over in the array?
```

Comment:
I would highly recommend utilizing console log in when you are not sure on what map is doing.  In this case, you can think of the first value in the map is the first object `{ name: "ford prefect", occupation: "a hitchhiker" }`, value 2 would be `{ name: "zaphod beeblebrox", occupation: "president of the galaxy" }`, and value 3 would be `{ name: "arthur dent", occupation: "a radio employee" }`